### PR TITLE
Feat/odex 43

### PR DESCRIPTION
**Pull request by quangdz1704**
What is purpose of the changes?

- Add Co harvest

### CODE REVIEW

**Review by sonnx-oraichain — APPROVED** [review]

File: src/pages/CoHarvest/components/BiddingHistory/index.tsx [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+} from 'pages/CoHarvest/hooks/useGetBidRound';
+import { useSimulate } from 'pages/UniversalSwap/SwapV3/hooks';
+import { memo, useEffect, useRef, useState } from 'react';
+import AllBidding from '../AllBidding';
+import MyBidding from '../MyBidding';
+import styles from './index.module.scss';
+
+const BiddingHistory = ({ round }) => {
+  const ORAIX_TOKEN_INFO = flattenTokens.find((e) => e.coinGeckoId === 'oraidex');
+  const USDC_TOKEN_INFO = flattenTokens.find((e) => e.coinGeckoId === 'usd-coin');
```
Comment (haunv3) [inline src/pages/CoHarvest/components/BiddingHistory/index.tsx:23]: need using oraichainTokens from oraidex-common

File: src/pages/CoHarvest/components/HarvestInfo/index.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  start: Date;
+  end: Date;
+  isStarted: boolean;
+}) => {
+  const { timeRemaining, percent, isEnd, poolValue, start, end, isStarted } = props;
+  const { data: prices } = useCoinGeckoPrices();
+  const [visible, setVisible] = useState(false);
+  const [theme] = useConfigReducer('theme');
+
+  const USDC_TOKEN_INFO = flattenTokens.find((e) => e.coinGeckoId === 'usd-coin');
```
Comment (haunv3) [inline src/pages/CoHarvest/components/HarvestInfo/index.tsx:28]: I think we can use oraichainTokens instead flattenTokens

File: src/pages/CoHarvest/hooks/useGetBidRound.ts [hunk 3]
```diff
@@ -0,0 +1,396 @@
...
+      if (!res.return_data[ind].success) {
+        return {
+          ...bid
+        };
+      }
+      const response = fromBinary(res.return_data[ind].data);
+      return response;
+    });
+
+    // const multicallPotential = await multicall.aggregate({
```
Comment (haunv3) [inline src/pages/CoHarvest/hooks/useGetBidRound.ts:340]: Please delete them if it is not necessary

**Review by haunv3 — APPROVED** [review]

**Review by haunv3 — APPROVED** [review]

**Review by trungbach — APPROVED** [review]
lgtm

**Pull request merged**